src/utils/superclusterConfig.ts

```diff
@@ -1,12 +1,15 @@
 import { ViewportProps } from '../features/common/types/map';
 import { Bound } from '../features/common/types/map';
-import Supercluster, { PointFeature } from 'supercluster';
+import Supercluster, { ClusterProperties, PointFeature } from 'supercluster';
 import { RefObject } from 'react';
 import { MapRef } from 'react-map-gl';
-import { DonationProperties } from '../features/common/types/myForestv2';
-import { MyContributionsSingleRegistration } from '../features/common/types/myForestv2';
+import {
+  DonationProperties,
+  MyContributionsSingleRegistration,
+} from '../features/common/types/myForestv2';
+import {} from '../features/common/Layout/MyForestContextV2';
 
-const clusterConfigV2 = {
+const _clusterConfigV2 = {
   radius: 40,
   maxZoom: 3,
 };
```

Comment:
When we use `_`, warnings for unused variables are suppressed as per our current eslint rules.